ONSdigital/dp-frontend-renderer
# Updated 'return to dataset' link to navigate to the current version rather than most recent version

### What

Updated the previewPage vendor (model and controller) to make a new field, `CurrentVersionURL`, available for the `preview-page` template. With this, the user can now navigate back to the current version dataset rather than most recent version's.

### How to review

- Pull changes in the `feature/current-version-url` for the `dp-frontend-models` and `dp-frontend-filter-dataset-controller` repositories.
- Navigate to a dataset on the website that has more than one version - select an older version.
- Click 'Filter and download' and then go through steps to apply filters
- Find text, 'Refer to the dataset page for more information...'
- Check that 'dataset page' hyperlink returns to the dataset version you were viewing.

### Who can review

Anyone
